projects/1.20/assets/moss-overgrowth/mossovergrowth/lang/zh_cn.json

```
@@ -0,0 +1,200 @@
+  "advancements.strateigic_planning.title": "战略策划",
+  "advancements.the_end_of_the_tunnel.descr": "有时，你不得不让自己成为那束光.",
+  "advancements.the_end_of_the_tunnel.title": "隧道尽头",
+  "advancements.the_king.descr": "所有的感染物都将跪倒在他们的新主人前!",
+  "advancements.the_king.title": "国王",
+  "advancements.the_moss_is_stalking_you.descr": "拿到了……？ 像是苔藓柄……?.. 我现在要走了...",
+  "advancements.the_moss_is_stalking_you.title": "苔藓缠绕着你",
+  "advancements.to_rule_them_all.descr": "每个指令与超级指令都至少有一个存在于你的物品栏中",
+  "advancements.to_rule_them_all.title": "统治一切……",
+  "advancements.you_shall_not_spread.descr": "制作传播阻断方块.",
+  "advancements.you_shall_not_spread.title": "你无法通过！",
+  "attribute.mossovergrowthregrown.cooldown": "Cooldown",
+  "attribute.mossovergrowthregrown.devolve": "转移",
+  "attribute.mossovergrowthregrown.evtimer": "计时器",
+  "attribute.mossovergrowthregrown.immune": "免疫",
+  "attribute.mossovergrowthregrown.immunevariants": "免疫变种",
+  "attribute.mossovergrowthregrown.kills": "击杀数",
+  "attribute.mossovergrowthregrown.mossmob_2": "苔藓怪物",
+  "attribute.mossovergrowthregrown.mossmobs": "苔藓怪物",
+  "attribute.mossovergrowthregrown.spreadspeed": "传播速度",
+  "attribute.mossovergrowthregrown.truemossmobs": "苔藓怪物",
+  "biome.mossovergrowthregrown.overgrowth": "苔藓爆发",
+  "block.mossovergrowthregrown.altered_obsidian": "被改造的黑曜石",
+  "block.mossovergrowthregrown.condensed_moss": "压缩苔藓",
+  "block.mossovergrowthregrown.moss_bubbler": "苔藓气泡方块",
+  "block.mossovergrowthregrown.moss_bulb": "苔藓杆",
+  "block.mossovergrowthregrown.moss_stalk": "苔藓杆",
+  "block.mossovergrowthregrown.mossy_brain": "苔藓大脑",
+  "block.mossovergrowthregrown.mossy_button": "苔藓按钮",
+  "block.mossovergrowthregrown.mossy_coal": "苔藓煤矿石",
+  "block.mossovergrowthregrown.mossy_copper": "苔藓红石铜",
+  "block.mossovergrowthregrown.mossy_core": "苔藓核心",
+  "block.mossovergrowthregrown.mossy_diamond": "苔藓钻石矿",
+  "block.mossovergrowthregrown.mossy_fence": "苔藓栅栏",
+  "block.mossovergrowthregrown.mossy_fence_gate": "苔藓栅栏们",
+  "block.mossovergrowthregrown.mossy_gold": "苔藓金矿石",
+  "block.mossovergrowthregrown.mossy_iron": "苔藓铁矿",
+  "block.mossovergrowthregrown.mossy_lapis": "苔藓青金石矿石",
+  "block.mossovergrowthregrown.mossy_leaves": "苔藓叶",
+  "block.mossovergrowthregrown.mossy_log": "苔藓原木",
+  "block.mossovergrowthregrown.mossy_planks": "苔藓木板",
+  "block.mossovergrowthregrown.mossy_pressure_plate": "苔藓压力板",
+  "block.mossovergrowthregrown.mossy_redstone": "苔藓红石矿",
+  "block.mossovergrowthregrown.mossy_slab": "苔藓台阶",
+  "block.mossovergrowthregrown.mossy_stairs": "苔藓台阶",
+  "block.mossovergrowthregrown.mossy_wood": "苔藓木",
+  "block.mossovergrowthregrown.plant_top": "苔藓平台",
+  "block.mossovergrowthregrown.spread_blocker": "传播阻断方块",
+  "block.mossovergrowthregrown.spread_blocker.description_0": "将其以四边形或封闭多边形的形状放置，来阻止感染物在范围中的传播",
+  "death.attack.moss": "%1$s 被自然吞噬",
+  "death.attack.moss.item": "%1$s 被 %2$s 用 %3$s 杀死了！",
+  "death.attack.moss.player": "%1$s 在尝试逃离 %2$s 时被大自然吞噬了",
```

```suggestion
  "death.attack.moss": "%1$s被大自然吞噬了",
  "death.attack.moss.item": "%1$s被%2$s用%3$s杀死了",
  "death.attack.moss.player": "%1$s在试图逃离%2$s时被大自然吞噬了",
```
统一格式，见https://zh.minecraft.wiki/w/%E6%AD%BB%E4%BA%A1%E6%B6%88%E6%81%AF